lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -427,59 +427,53 @@ def creds_search(*args)
       'SearchTerm' => search_term
     }
 
-    tbl = Rex::Text::Table.new(tbl_opts)
+
```

Comment:
```suggestion
```
Extra line left over from deletion?

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/15623/commits/ccccccde27b84d6593c8146315d79ee1852cdb10